core/src/cg/group4/rewards/CollectionUtil.java

```diff
@@ -0,0 +1,49 @@
+package cg.group4.rewards;
+
+import cg.group4.exceptions.LocalStoreUnavailableException;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.files.FileHandle;
+
+/**
+ * Utility class for collection.
+ */
+public final class CollectionUtil {
```

Comment:
See class seems over kill, all it does is call gdx methods and does a single check.